src/bin/after_success.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'"
+echo "TRAVIS_PULL_REQUEST='$TRAVIS_PULL_REQUEST'"
+
+# For now, obfuscate SNAPSHOTs from sbt's developers: https://github.com/sbt/sbt/issues/2687#issuecomment-236586241
+if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    if [ "$TRAVIS_BRANCH" == "develop" ]; then
+        sbt \
+            'set test in Test := {}' \
+            'set isSnapshot := false' \
+            'set resolvers += Resolver.url("bintray-sbt-plugin-releases", url("http://dl.bintray.com/content/sbt/sbt-plugin-releases"))(Resolver.ivyStylePatterns)' \
+            'set publishTo := Option("artifactory-publish" at "https://artifactory.broadinstitute.org/artifactory/libs-release-local;build.timestamp=" + new java.util.Date().getTime)' \
+            "set credentials += Credentials(\"Artifactory Realm\", \"artifactory.broadinstitute.org\", \"${ARTIFACTORY_USERNAME}\", \"${ARTIFACTORY_PASSWORD}\")" \
+            publish 2>&1 | sed 's/.*set credentials.*/REDACTED LOG LINE/'
+
+    elif [[ "$TRAVIS_BRANCH" =~ ^[0-9\.]+_hotfix$ ]]; then
+        sbt \
+            'set test in Test := {}' \
+            'set isSnapshot := false' \
+            'set git.gitUncommittedChanges := false' \
```

Comment:
There really should be more cross commenting between the two, but for now...

This is tightly linked to hoops we jump through in our build.sbt. Over in build.sbt, for the default behavior when someone is building, we set this flag to `true`. That triggers the git hashing plugin to append `-SNAP`. Overriding the flag to false here unsets that appending behavior.